src/player.cpp

```diff
@@ -9518,11 +9518,14 @@ void player::do_read( item *book )
                                           pgettext( "memorial_female", "Reached skill level %1$d in %2$s." ),
                                           ( int )skill_level, skill.obj().name().c_str() );
                     }
-                    // possible callback arguments: skill_increase_type, skill_id, skill_value_prev, skill_value_new
-                    lua_callback( "on_skill_increased", "book" );
+                    // possible callback arguments: skill_increase_type, skill_id, skill_level_new
+                    const static auto skill_increase_type = "book";
+                    const static auto skill_id = skill.c_str();
+                    const static auto skill_level_new = std::to_string( originalSkillLevel + 1 ).c_str();
```

Comment:
Same here. Btw. please use `to_string` without the `std::` prefix. See "src/compatibility.h", some systems seem to have trouble with `std::to_string` (it is called without the prefix throughout the code).